src/layouts/CreateProjectLayout.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from "react";
+import Grid from "@material-ui/core/Grid";
+import styled from "styled-components";
+
+import ProgressBar from "../../src/components/general/ProgressBar.js";
+import ProjectPaneContents from "../../src/components/project/ProjectPaneContents.js";
+import Button from "../../src/components/general/Button";
+import TEMP_ORGANIZATION_DATA from "./../../tmp-data/organizations.json";
+import TEMP_CATEGORY_DATA from "./../../tmp-data/categories.json";
+
+class CreateProjectLayout extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      organizations: TEMP_ORGANIZATION_DATA.organizations,
+      categories: TEMP_CATEGORY_DATA.categories,
+      progressPercentage: 0,
+      organization_id: "",
```

Comment:
JavaScript identifiers are typically `camelCased`. Is there a reason we're using underscores here?